### PR TITLE
x509-idenity-mgmt component documentation

### PR DESCRIPTION
**Pull request by thiagodpf**
A brief description of the x509-idenity-mgmt component has been included in place of the old EJBCA REST

dojot/dojot#1473

### CODE REVIEW

File: source/components-and-apis.rst [hunk 1]
```diff
@@ -355,12 +341,11 @@ dojot.
...
-.. _API - EJBCA-REST: https://dojot.github.io/ejbca-rest/apiary_v0.4.3.html
-
 .. _GitHub - Data Manager: https://github.com/dojot/data-manager
 .. _API - Data Manager: https://dojot.github.io/data-manager/apiary_v0.4.3.html
 
 .. _GitHub - Cron: https://github.com/dojot/cron
 .. _API - Cron: https://dojot.github.io/cron/apiary_v0.4.3.html
 
+.. _GitHub - x509-identity-mgmt: https://github.com/dojot/dojot/tree/development/x509-identity-mgmt
+.. _API - x509-identity-mgmt: https://dojot.github.io/dojot/x509-identity-mgmt/apiary_release-v0.5.0-alpha.html
```
Comment (mprevide) [inline source/components-and-apis.rst:351]: The final link should be https://dojot.github.io/dojot/x509-identity-mgmt/apiary_v0.5.0.html

File: source/components-and-apis.rst [hunk 2]
```diff
@@ -355,12 +341,11 @@ dojot.
...
-.. _GitHub - EJBCA-REST: https://github.com/dojot/ejbca-rest
-.. _API - EJBCA-REST: https://dojot.github.io/ejbca-rest/apiary_v0.4.3.html
-
 .. _GitHub - Data Manager: https://github.com/dojot/data-manager
 .. _API - Data Manager: https://dojot.github.io/data-manager/apiary_v0.4.3.html
 
 .. _GitHub - Cron: https://github.com/dojot/cron
 .. _API - Cron: https://dojot.github.io/cron/apiary_v0.4.3.html
 
+.. _GitHub - x509-identity-mgmt: https://github.com/dojot/dojot/tree/development/x509-identity-mgmt
```
Comment (mprevide) [inline source/components-and-apis.rst:350]: The final link should be https://github.com/dojot/dojot/tree/master/x509-identity-mgmt

File: source/components-and-apis.rst [hunk 3]
```diff
@@ -286,6 +269,9 @@ TCP port 8000), see the following table.
    * - Data Broker
      - host:80/socket.io
      - host:8000/socket.io
+   * - X.509 Identity Management
+     - host:80/x509
+     - host:8000/x509
```
Comment (mprevide) [inline source/components-and-apis.rst:274]: host:8000/x509/v1?

File: source/components-and-apis.rst [hunk 4]
```diff
@@ -286,6 +269,9 @@ TCP port 8000), see the following table.
    * - Data Broker
      - host:80/socket.io
      - host:8000/socket.io
+   * - X.509 Identity Management
+     - host:80/x509
```
Comment (mprevide) [inline source/components-and-apis.rst:273]: host:80?

File: source/components-and-apis.rst [hunk 5]
```diff
@@ -215,6 +199,11 @@ See more about how using API in :ref:`Using API interface`.
      - /socket.io
      - `API - data-broker`_
      - No
+   * - X.509 Identity Management
+     - /x509
```
Comment (mprevide) [inline source/components-and-apis.rst:203]: /?

File: source/components-and-apis.rst [hunk 6]
```diff
@@ -215,6 +199,11 @@ See more about how using API in :ref:`Using API interface`.
...
+     - /api
```
Comment (mprevide) [inline source/components-and-apis.rst:204]: /x509/v1?

File: source/components-and-apis.rst [hunk 7]
```diff
@@ -256,12 +245,6 @@ TCP port 8000), see the following table.
    * - History
      - host:8000/
      - host:8000/history
-   * - EJBCA REST
-     - host:5583/sign
-     - host:8000/sign
-   * - EJBCA REST
-     - host:5583/ca
-     - host:8000/ca
    * - Data Manager
```
Comment (mprevide) [inline source/components-and-apis.rst:248]: Please take the opportunity and fix it:

   * - Auth
     - host:5000/revoke
     - host:8000/auth/revoke

File: source/components-and-apis.rst (hunk 6 above)
Comment (thiagodpf) [reply source/components-and-apis.rst:204]: Currently it is `/api/v1` but I think I will end up changing to be in the same mold as kafka-ws (next release).

File: source/locale/pt_BR/LC_MESSAGES/concepts.po [hunk 8]
```diff
@@ -124,11 +124,83 @@ msgstr ""
 "pode encontrar um tutorial de como autenticar um usuário e obter um token"
 " de acesso na `auth documentation`_."
 
-#: ../../source/concepts.rst:56
+#: ../../source/concepts.rst:51
+#, fuzzy
```
Comment (mprevide) [inline source/locale/pt_BR/LC_MESSAGES/concepts.po:128]: When has `#, fuzzy`, so probably has a problem.
You can fix and remove the line with the fuzzy

Comment (thiagodpf) [reply source/locale/pt_BR/LC_MESSAGES/concepts.po:128]: You're right, `gettext` tried to infer a translation automatically ...

File: source/locale/pt_BR/LC_MESSAGES/concepts.po [hunk 9]
```diff
@@ -124,11 +124,82 @@ msgstr ""
...
+"um canal de comunicação seguro com a plataforma dojot, na qual o dispositivo "
+"é identificado através do seu certificado."
+
+#: ../../source/concepts.rst:76
+msgid ""
+"After the secure channel is established, the device is able to publish "
+"data and also receive data from the dojot platform as long as it is "
+"authorized to do so."
+msgstr ""
+"Após o canal seguro estabelecido, o dispositivo é capaz de publicar dados e "
```
Comment (eduardogmisiuk) [inline source/locale/pt_BR/LC_MESSAGES/concepts.po:194]: Após o canal seguro ser estabelecido, o dispositivo é capaz de publicar e

File: source/locale/pt_BR/LC_MESSAGES/concepts.po [hunk 10]
```diff
@@ -124,11 +124,82 @@ msgstr ""
...
+"também receber dados da plataforma dojot desde que ele esteja autorizado a "
```
Comment (eduardogmisiuk) [inline source/locale/pt_BR/LC_MESSAGES/concepts.po:195]: receber dados [...]

File: source/locale/pt_BR/LC_MESSAGES/internal-communication.po [hunk 11]
```diff
@@ -617,6 +617,33 @@ msgstr ""
...
+msgstr ""
+"A plataforma dojot possui internamente uma autoridade certificadora (`CA`_) "
+"capaz de emitir certificados x.509 para que os dispositivos possam se "
+"comunicar com a plataforma através de um canal seguro (usando o protocolo "
+"TLS). Ao requisitar um certificado para a plataforma, é necessário informar "
+"um `CSR`_, o qual passará por uma série de validações até chegar na "
+"Autoridade Certificadora interna, que por sua vez, se todas as verificações "
+"passarem com sucesso, assinará um certificado e vinculará este certificado "
+"ao registro do dispositivo. O componente ``x509-identity-mgmt`` é responsável "
+"por oferecer os serviços relacionados à certificados para dispositivos."
```
Comment (eduardogmisiuk) [inline source/locale/pt_BR/LC_MESSAGES/internal-communication.po:644]: à -> a

**Pull request merged**